hknweb/events/views.py

```diff
@@ -306,3 +306,22 @@ def send_off_waitlist_email(request, user, event):
                 settings.NO_REPLY_EMAIL, [user.email])
     msg.attach_alternative(html_content, "text/html")
     msg.send()
+
+
+# Leaderboard
+def show_leaderboard(request):
+    users = [
```

Comment:
TODO: Use real users and have a way to calculate points in the future

(Hint: Consider the "User" Django table)